src/test/regress/expected/create_role_propagation.out

```diff
@@ -259,7 +259,25 @@ SELECT result FROM run_command_on_all_nodes(
   {"member":"test_admin_role","role":"dist_role_3","grantor":"postgres","admin_option":true}]
 (3 rows)
 
-REVOKE dist_role_3 from dist_role_4 granted by test_admin_role;
+REVOKE dist_role_3 from dist_role_4 granted by test_admin_role cascade;
+SELECT result FROM run_command_on_all_nodes(
+  $$
+  SELECT json_agg(q.* ORDER BY member) FROM (
+    SELECT member::regrole::text, roleid::regrole::text AS role, grantor::regrole::text, admin_option
+    FROM pg_auth_members WHERE roleid::regrole::text = 'dist_role_3'
+  ) q;
+  $$
+);
+                                                result
+---------------------------------------------------------------------
+ [{"member":"non_dist_role_3","role":"dist_role_3","grantor":"test_admin_role","admin_option":false}, +
+  {"member":"test_admin_role","role":"dist_role_3","grantor":"postgres","admin_option":true}]
+ [{"member":"dist_role_4","role":"dist_role_3","grantor":"test_admin_role","admin_option":false},     +
```

Comment:
The following line disappeared from coordinator (see the first array in `result`) and but not from workers, do you think what's going wrong?

```sql
{"member":"dist_role_4","role":"dist_role_3","grantor":"test_admin_role","admin_option":false}
```

Reply:
Also, flaky test detection jobs are failing.

Reply:
Yes I noticed right now. It is successful in local. Let me check it

Reply:
> The following line disappeared from coordinator (see the first array in `result`) and but not from workers, do you think what's going wrong?
> 
> ```sql
> {"member":"dist_role_4","role":"dist_role_3","grantor":"test_admin_role","admin_option":false}
> ```

Disappearing from coordinator's output (first array in the `result`) is the correct behavior but the fact that this doesn't apply to worker outputs doesn't sound correct.

Reply:
Problem was I didn't perform a fresh build before getting test outputs. Now pushed test outputs with fresh builds